sdk/src/main/java/com/sygic/travel/sdk/utils/responseExtensionFunctions.kt

```diff
@@ -0,0 +1,14 @@
+package com.sygic.travel.sdk.utils
+
+import android.util.Log
+import retrofit2.Call
+
+
+internal fun <T : Any> Call<T>.getBody(): T? {
```

Comment:
this commit is probably totally useless..... what are you trying to achieve?
we should be clear when something happens and do not hide the exceptions.